test/test_sparse_ops.py

```diff
@@ -0,0 +1,367 @@
+import math
+from itertools import product, chain
+from numbers import Number
+import random
+
+import unittest
+
+import torch
+
+from torch.testing._internal.common_utils import \
+    (TestCase, run_tests, torch_to_numpy_dtype_dict, suppress_warnings,
+     TEST_NUMPY, IS_MACOS, make_tensor)
+from torch.testing._internal.common_methods_invocations import \
+    (sparse_unary_ufuncs)
+from torch.testing._internal.common_device_type import \
+    (instantiate_device_type_tests, ops, dtypes)
+from torch.testing import \
+    (floating_types_and, integral_types, all_types_and_complex_and)
+
+if TEST_NUMPY:
```

Comment:
We require NumPy now so you can just `import numpy as np` unconditionally

Reply:
Made the changes as per the suggestion